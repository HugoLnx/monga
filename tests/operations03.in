int main() {
	x = -a * b;
}
