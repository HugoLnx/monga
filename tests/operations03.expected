TK_MINUS exp
exp TK_ASTERISK exp
var TK_ONE_EQUAL exp
